src/views/ToDoList/ToDoList.test.tsx

```diff
@@ -0,0 +1,262 @@
+import { vi, Mock } from 'vitest';
+import { Task } from '../../shared/types/Task';
+import { toDosApi } from '../../shared/api/toDosApi';
+import { ToDoList } from './index';
+import { fireEvent, render } from '@testing-library/react';
+
+vi.mock('../../shared/api/toDosApi', () => {
+  return {
+    toDosApi: {
+      fetchToDos: vi.fn(),
+      deleteTask: vi.fn(),
+      createNewTask: vi.fn(),
+    },
+  };
+});
+
+describe('The ToDoList', () => {
+  let newTaskTitle: string;
+  let newTaskMock: Task;
+  let task1: Task;
```

Comment:
Four variables, such as `task1` and `task2`, do not seem very elegant. Maybe try using an array instead?